target/linux/ramips/image/Makefile

```
@@ -218,7 +218,14 @@ endef
 
 define Device/uimage-lzma-loader
   LOADER_TYPE := bin
-  KERNEL := kernel-bin | append-dtb | lzma | loader-kernel | uImage none
+  KERNEL/lzma-loader := kernel-bin | append-dtb | lzma | loader-kernel
+  KERNEL := $$(KERNEL/lzma-loader) | uImage none
+endef
+
+define Device/seama-lzma-loader
+  $(Device/seama)
+  $(Device/uimage-lzma-loader)
+  KERNEL := $$(KERNEL/lzma-loader) | lzma -a0
```

DIR-860L recipe currently uses:
* `relocate-kernel` and
*  `uImage lzma`.

It surely doesn't work without `reloacate-kernel`. It may work without `uImage lzma`. I found the following in my mailbox:
```
About the lzma repack: I found out that the U-Boot accepts
uncompressed and lzma compressed initramfs images.
It doesn't accepts gzip initramfs, but I didn't tested other compressions.
I also found that the U-Boot expects lzma compressed kernel images
from the flash and doesn't care about uImage header.
```

The DIR-645:
* doesn't care about `relocate-kernel`: it boots with and without it :+1: 
* doesn't boot with `uImage lzma` header


The following could work for both:
```suggestion
  KERNEL := $$(KERNEL/lzma-loader) | relocate -kernel | lzma -a0
```

---

DIR-860L's `uImage lzma` is maybe for initramfs. :thinking: